be/src/exec/vectorized/analytor.h

```diff
@@ -124,7 +124,7 @@ class Analytor final : public pipeline::ContextWithDependency {
     RuntimeState* _state = nullptr;
     bool _is_closed = false;
 
-    const TPlanNode& _tnode;
+    const TPlanNode _tnode;
```

Comment:
Why change it?

Reply:
The _tnode is only valid in the init and prepare phases and will be destructured after the prepare phase

Reply:
Why after  prepare phases，you still use it?

Reply:
I need to use some properties of the TFunction such as URL checksum, symbol to get the function and load the corresponding class